src/event/event_kevent.c

```diff
@@ -32,6 +32,14 @@
 #define DISPATCH_KEVENT_MUXED_MARKER  1ul
 #define DISPATCH_MACH_AUDIT_TOKEN_PID (5)
 
+#if defined(__FreeBSD__)
+typedef void* dispatch_kqueue_udata;
```

Comment:
The name is wrong, udata is a kevent field, not udata. `dispatch_kevent_udata_t` would make more sense, and define it once this way:


```C
#define dispatch_kevent_udata_t  typeof(((dispatch_kevent_t)NULL)->udata)
```